packages/server/beaver_triple_service/utils/interceptor.go

```diff
@@ -75,14 +74,6 @@ func AuthJWT(tokenString string) (*jwt_types.Claim, error) {
 		return nil, status.Error(codes.Internal, "failed claims type assertions")
 	}
 
-	// expのバリデーション
-	if claims, ok := token.Claims.(jwt.MapClaims); ok && token.Valid {
-		exp := time.Unix(int64(claims["exp"].(float64)), 0)
-		if time.Now().After(exp) {
-			return nil, status.Error(codes.Internal, "token is expired")
-		}
-	}
```

Comment:
I deleted this because it was validated with `jwt.ParseWithClaims` and did not reach here 
https://github.com/golang-jwt/jwt/blob/b357385d3ee5c53c16db81addf9c6c8059c4cf15/parser.go#L105
https://github.com/golang-jwt/jwt/blob/b357385d3ee5c53c16db81addf9c6c8059c4cf15/validator.go#L90

Reply:
Oh...
Thanks for noticing and correcting my mistake.